src/crab/wto.cpp

```diff
@@ -55,7 +55,7 @@ void wto_t::start_visit(const label_t& vertex, wto_partition_t& partition, std::
 
     if (head_dfn == vertex_data.dfn) {  
         vertex_data.dfn = INT_MAX;
-        label_t& element = _stack.top();
+        label_t element = _stack.top();
```

Comment:
Fix for #303

Reply:
Either move this change to a separate PR or update the description of this PR to say it also fixes 303.

Reply:
Will update the description. This is required to make it work with line_info move to label_t (it crashed otherwise).